app/src/main/java/uk/gov/android/securestore/SharedPrefsStore.kt

```diff
@@ -181,16 +187,20 @@ class SharedPrefsStore(
     }
 
     private fun cryptoDecryptText(
-        key: String,
+        alias: String,
         onTextReady: (String?) -> Unit,
     ) {
         sharedPrefs?.let {
             try {
-                it.getString(key, null)?.let { encryptedText ->
-                    cryptoManager.decryptText(
-                        encryptedText,
-                    ) { text -> onTextReady(text) }
-                } ?: onTextReady(null)
+                val encryptedData = it.getString(alias, null)
+                val encryptedKey = it.getString(alias + "Key", null)
```

Comment:
Think we should probably have the `"Key"` appendage created as a constant (maybe in the `HybridCryptoManager`)

Reply:
As in a function as a companion that creates it?